lib/board.rb

```diff
@@ -0,0 +1,19 @@
+# Understands how to represent a board for tic-tac-toe
+
+class Board
+  attr_reader :grid
+
+  def initialize
+    @grid = Array.new(3) { Array.new(3) { :" " } }
+  end
+
+  def fill_at(index, player)
+    grid[index[Parser::ROW_POSITION]][index[Parser::COL_POSITION]] = player.token
+  end
+
+  def display
+    puts "|#{grid[0][0]}" + "|#{grid[0][1]}" + "|#{grid[0][2]}|"
```

Comment:
Encapsulation: It feels like a missing domain concept is necessitating a bunch of interrogation of state and violation of encapsulation.